src/Model/Booking.php

```diff
@@ -303,7 +305,9 @@ public function returnDatetime(): string {
 			return $date_end;
 		}
 
-		if ( $this->getGrid() === 0 ) { // if grid is set to slot duration
+		$grid = $this->getGrid();
```

Comment:
Das war nur zum testen da, oder?

Reply:
@hansmorb ne das wird in Zeile 314, 318 und 319 genutzt. Es gibt Fälle in denen `$grid` undefined ist.